fuse_constraints/include/fuse_constraints/relative_pose_2d_stamped_constraint.h

```diff
@@ -65,20 +66,36 @@ class RelativePose2DStampedConstraint : public fuse_core::Constraint
   /**
    * @brief Constructor
    *
-   * @param[in] position1    The variable representing the position components of the first pose
-   * @param[in] orientation1 The variable representing the orientation components of the first pose
-   * @param[in] position2    The variable representing the position components of the second pose
-   * @param[in] orientation2 The variable representing the orientation components of the second pose
-   * @param[in] delta        The measured change in the pose (3x1 vector: dx, dy, dyaw)
-   * @param[in] covariance   The measurement covariance (3x3 matrix: dx, dy, dyaw)
+   * Note that, when measuring subset of dimensions, empty axis vectors are permitted. This signifies, e.g., that you
```

Comment:
Not sure `e.g.` is needed here
```suggestion
   * Note that, when measuring subset of dimensions, empty axis vectors are permitted. This signifies that you
```